js/main.js

```diff
@@ -135,3 +134,136 @@ function bigPictureInit(photo) {
 }
 
 bigPictureInit(photos[0]);
+
+
+const modalOnload = document.querySelector(`#upload-file`);
+const modalOpen = document.querySelector(`.img-upload__overlay`);
+const modalClose = document.querySelector(`#upload-cancel`);
```

Comment:
```suggestion
const modalOnloadElement = document.querySelector(`#upload-file`);
const modalOpenElement = document.querySelector(`.img-upload__overlay`);
const modalCloseElement = document.querySelector(`#upload-cancel`);
```

Но что то мне подсказывает что имена для переменных выбраны не очень хорошо